onnx/defs/operator_sets.h

```diff
@@ -533,6 +533,7 @@ class ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 10, IsInf);
 class ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 10, NonMaxSuppression);
 class ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 10, ReverseSequence);
 class ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 10, RoiAlign);
+class ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Onnx, 10, Loop);
```

Comment:
This should be opset 11, since we already release 1.5. 10 should be frozen.

Reply:
Thanks for catching this. I have updated it. (I created this PR a while back when it was still opset 10 and forgot about it now.)